stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/MessageComposerDefaultMentionSuggestionsContent.kt

```diff
@@ -0,0 +1,137 @@
+package io.getstream.chat.android.ui.message.composer
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import androidx.core.view.isVisible
+import androidx.recyclerview.widget.RecyclerView
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.common.state.MessageInputState
+import io.getstream.chat.android.ui.R
+import io.getstream.chat.android.ui.common.extensions.internal.streamThemeInflater
+import io.getstream.chat.android.ui.databinding.StreamUiItemMentionBinding
+import io.getstream.chat.android.ui.databinding.StreamUiSuggestionListViewBinding
+
+/**
+ * Default implementation of mention suggestions view. Displayed in a popup above [MessageComposerView] when there are
+ * mention suggestions available in the current [MessageInputState].
+ */
+internal class DefaultMentionSuggestionsContent : FrameLayout, MessageComposerChild {
+
+    /**
+     * Handle to layout binding.
+     */
+    private val binding: StreamUiSuggestionListViewBinding =
+        StreamUiSuggestionListViewBinding.inflate(streamThemeInflater, this)
+
+    /**
+     * Callback invoked when mention suggestion is selected.
+     */
+    public var onMentionSelected: (User) -> Unit = {}
+
+    /**
+     * Adapter used to render mention suggestions.
+     */
+    private val adapter = MentionsAdapter { onMentionSelected(it) }
+
+    constructor(context: Context) : this(context, null)
+
+    constructor(context: Context, attrs: AttributeSet?) : this(context, attrs, 0)
+
+    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context,
+        attrs,
+        defStyleAttr) {
+        init()
+    }
+
+    /**
+     * Set up initial layout state.
+     */
+    private fun init() {
+        binding.apply {
+            suggestionsCardView.isVisible = true
+            commandsTitleTextView.isVisible = false
+            suggestionsRecyclerView.adapter = adapter
+        }
+    }
+
+    /**
+     * Propagates list of currently available mention suggestions to adapter.
+     */
+    override fun renderState(state: MessageInputState) {
+        adapter.setMentions(state.mentionSuggestions)
+    }
+}
+
+/**
+ * [RecyclerView.Adapter] responsible for displaying mention suggestions in the RecyclerView in [DefaultMentionSuggestionsContent].
+ */
+internal class MentionsAdapter(inline val onMentionSelected: (User) -> Unit) : RecyclerView.Adapter<MentionsViewHolder>() {
```

Comment:
Should we maybe pull this out in separate files? We generally have that for Compose, so that it's easier to find all the classes and things in the project :]